cron/k8s/transfer.release-raw.yaml

```diff
@@ -33,7 +33,7 @@ spec:
             - name: RAW_SCORECARD_DATA_BUCKET_URL
               value: "gs://ossf-scorecard-rawdata-releasetest"
             - name: RAW_SCORECARD_BIGQUERY_TABLE
-              value: "scorecard_raw-releasetest"
+              value: "scorecard_rawdata-releasetest"
```

Comment:
If BQ table is not already created, maybe rename to `scorecard-rawdata-releasetest` to maintain convention? Feel free to ignore.

Reply:
I like yours better, but the other table is called `scorecard_releasedataset` (which is why I named it this way). More than happy to update, though

Reply:
Upto you. If it's not a lot of work to change the table name, can use the existing convention.